frontend/scripts/pages/profile-actor.page.js

```diff
@@ -325,15 +328,12 @@ const _init = ()  => {
 
   fetch(actorFactsheetURL)
     .then((response) => {
-      if (response.status === 404) {
-        _showErrorMessage();
-        return null;
-      }
-
-      if (response.status === 200) {
+      if (response.ok) {
         return response.json();
       }
-    })
+      _showErrorMessage(response.statusText);
+    }, (reason) => _showErrorMessage(reason.message)
```

Comment:
Nifty af! I didn't know of the existence of Promise.then second argument. 🙌 
As a side note, I prefer using `.catch` for handling errors but this is perfectly fine.